usingcurl-smtp.md

```diff
@@ -60,12 +60,14 @@ adding `--ssl` to the command:
 
     curl --ssl smtp://mail.example.com --mail-from myself@example.com
          --mail-rcpt receiver@example.com --upload-file email.txt
+         --user 'user@your-account.com:your-accout-password'
```

Comment:
misspelled the second `account`, also in the next edit...

Reply:
@bagder done